package.json

```diff
@@ -1,65 +1,87 @@
 {
-	"name": "electric-io",
-	"version": "0.1.0",
-	"description": "an IoT Dashboard for Azure IoT Hub",
-	"author": "Suz Hinton",
-	"main": "index.js",
-	"scripts": {
-		"start": "npm run watch",
-		"watch": "npm-run-all --parallel watch:server watch:build",
-		"watch:build": "webpack --progress --colors --watch",
-		"watch:server": "nodemon index.js",
-		"test": "jest",
-		"test:watch": "jest --watchAll"
-	},
-	"dependencies": {
-		"azure-iothub": "^1.2.5",
-		"azure-iothub-receiver": "^2.0.0",
-		"dotenv": "^5.0.0",
-		"express": "^4.12.4",
-		"socket.io": "*"
-	},
-	"engines": {
-		"node": ">= 6"
-	},
-	"engineStrict": true,
-	"keywords": [
-		"node",
-		"express",
-		"iot"
-	],
-	"license": "MIT",
-	"devDependencies": {
-		"@vue/test-utils": "^1.0.0-beta.18",
-		"babel-cli": "^6.26.0",
-		"babel-core": "^6.26.0",
-		"babel-jest": "^22.4.3",
-		"babel-loader": "^7.1.4",
-		"babel-minify-webpack-plugin": "^0.3.1",
-		"babel-preset-env": "^1.6.1",
-		"debug": "^3.1.0",
-		"jest": "^22.4.3",
-		"nodemon": "^1.17.3",
-		"npm-run-all": "^4.1.2",
-		"vue": "^2.5.16",
-		"vue-template-compiler": "^2.5.16",
-		"webpack": "^4.5.0",
-		"webpack-cli": "^2.0.14",
-		"webpack-watch-server": "^1.2.1"
-	},
-	"jest": {
-		"collectCoverage": true,
-		"collectCoverageFrom": [
-			"lib/**/*.js",
-			"public/js/**/*.js",
-			"!public/js/{dist,vendor}/*.js"
-		],
-		"roots": [
-			"lib",
-			"public/js"
-		],
-		"setupFiles": [
-			"./lib/testSetup.js"
-		]
-	}
+  "name": "electric-io",
+  "version": "0.1.0",
+  "description": "an IoT Dashboard for Azure IoT Hub",
+  "author": "Suz Hinton",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run watch",
+    "watch": "npm-run-all --parallel watch:server watch:build",
+    "watch:build": "webpack --progress --colors --watch",
+    "watch:server": "nodemon index.js",
+    "test": "jest",
+    "test:watch": "jest --watchAll"
+  },
+  "dependencies": {
+    "azure-iothub": "^1.2.5",
```

Comment:
We don't have to change this, but I'm a bit leery of having carrot-prefix on dependency numbers. I know that's the default when you `npm install` things, but... bad things can happen sometimes.

Reply:
I totally agree. I'll lockdown versioning a bit better before this gets merged.

Reply:
do we think this kind of change belongs in this PR? caret deps exist prior to this. I feel like we're good to merge.

Reply:
Agreed, I'll put up another PR in the next day or so to address the versioning in this file.